Cargo.toml

```diff
@@ -15,6 +15,9 @@ indicatif = "0.9"
 statistical = "0.1"
 atty = "0.2.2"
 cfg-if = "0.1.2"
+csv = "1.0.0-beta.5"
```

Comment:
Do we really need to rely on a beta version here? Would the latest stable version work, too?

Reply:
I'll check, our use case is pretty simple so I'm hopeful older versions would work

Reply:
It appears that serde support is not available in pre 1.0.0 versions. I tried 0.15.0 but it is requiring some rather significant changes to serialization, writing, and so on. Not to mention that I can no longer find all the documentation for that version (Encodable trait results in 404). It may be reasonable to use the beta for now, which will also allow keeping serde support betwen json and csv

Reply:
Okay, thanks for checking!